src/new-client/src/services/connection.ts

```diff
@@ -86,12 +92,53 @@ const idleDisconnect$: Observable<ConnectionStatus> = combineLatest([
   }),
 )
 
+const online$: Observable<Boolean> = merge(
+  of(navigator.onLine),
+  fromEvent(window, "online").pipe(mapTo(true)),
+  fromEvent(window, "offline").pipe(mapTo(false)),
+).pipe(
```

Comment:
Minor - This pipe isn't necessary

Reply:
Removed the pipe.

Also added comments above $online to be clear on Network connection vs Internet connection